main.py

```diff
@@ -670,6 +670,42 @@ def delete_link_metadata(self, request: Request) -> JSONResponse:
         self.notify_topology_update()
         return JSONResponse("Operation successful")
 
+    @rest('v3/links/{link_id}', methods=['DELETE'])
+    def delete_link(self, request: Request) -> JSONResponse:
+        """Delete a disabled link from topology.
+         It won't work for link with other statuses.
+        """
+        link_id = request.path_params["link_id"]
+        try:
+            with self._links_lock:
+                link = self.links[link_id]
+                if link.status != EntityStatus.DISABLED:
+                    raise HTTPException(409, detail="Link is not disabled.")
+                link = self.links.pop(link_id)
+                switch_a = link.endpoint_a.switch
+                switch_b = link.endpoint_b.switch
+                if link.endpoint_a.link and link == link.endpoint_a.link:
+                    link.endpoint_a.link = None
+                    link.endpoint_a.nni = False
+                if link.endpoint_a.link and link == link.endpoint_b.link:
+                    link.endpoint_b.link = None
+                    link.endpoint_b.nni = False
+        except KeyError:
+            raise HTTPException(404, detail="Link not found.")
+
+        self.topo_controller.upsert_switch(switch_a.id, switch_a.as_dict())
+        if switch_a != switch_b:
+            self.topo_controller.upsert_switch(
+                switch_b.id, switch_b.as_dict()
+            )
+        self.topo_controller.delete_link(link_id)
+        self.notify_topology_update()
+        name = 'kytos/topology.link_down'
+        content = {'link': link, 'reason': "link disabled"}
+        event = KytosEvent(name=name, content=content)
+        self.controller.buffers.app.put(event)
```

Comment:
The event notification looks good, but I'll recommend you use `self.notify_link_status_change(link, reason="link disabled") instead to reuse the existing method, which also has another lock protection for link notifications.